scripts/auto_noise_run.py

```diff
@@ -6,31 +6,31 @@
 from hera_pspec import utils
 from hera_pspec.container import PSpecContainer
 from pyuvdata import UVData
+from hera_cal._cli_tools import parse_args, run_with_profiling
 
-parser = utils.uvp_noise_error_parser()
-args = parser.parse_args()
-
-# compute Tsys from autocorr uvd
-uvd = UVData()
-uvd.read(args.auto_file)
-auto_Tsys = utils.uvd_to_Tsys(uvd, beam=args.beam)
-# load in pspec container.
-psc = PSpecContainer(args.pspec_container, keep_open=False, mode='rw', swmr=False)
+def main(args):
```

Comment:
That's the previous PR, right? it's probably worth merging `main` to have a clean new PR